server_maintain/__init__.py

```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
```

Comment:
name of the module should be reviewed as well as it is not obvious to understand the meaning